assignments/array-methods.js

```diff
@@ -458,31 +458,48 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
 let fullNames = [];
-runners.forEach(function(element) {
-	console.log(element);
+const names = runners.forEach(element => {
+	return fullNames.push(`${element.first_name} ${element.last_name}`);
 });
 console.log(fullNames);
```

Comment:
no need for a return statement inside a forEach.  map loops through an array and returns a new array and forEach just loops through an array